core/src/main/scala/org/broadinstitute/dsde/rawls/billing/GoogleBillingProjectCreator.scala

```diff
@@ -46,9 +37,10 @@ class GoogleBillingProjectCreator(samDAO: SamDAO, gcsDAO: GoogleServicesDAO)(imp
     } yield {}
 
   override def postCreationSteps(createProjectRequest: CreateRawlsV2BillingProjectFullRequest,
+                                 config: MultiCloudWorkspaceConfig,
                                  ctx: RawlsRequestContext
-  ): Future[Unit] =
+  ): Future[CreationStatus] =
     for {
       _ <- syncBillingProjectOwnerPolicyToGoogleAndGetEmail(samDAO, createProjectRequest.projectName)
-    } yield {}
+    } yield { CreationStatuses.Ready }
```

Comment:
Changing this to return the status makes a lot of sense here - I'm assuming that for the Azure billing projects, where we need to create landing zones, it will  return `CreationStatuses.Creating`?

Reply:
Actually the new `CreationStatuses. CreatingLandingZone` value. But although I like returning the status here, the reality is that the BPM implementation still needs to update a database to insert a record into the `WorkspaceManagerResourceMonitorRecord` table. Therefore I'm wondering if I should also just have it deal with updating the status itself and leave the Google implementation alone.

Reply:
I decided to keep this returning the creation status because we want to set it initially to `Creating` (to avoid going from `Ready` to `CreatingLandingZone`).